cxf-codegen-gradle/src/main/java/io/mateo/cxf/codegen/wsdl2java/WsdlOption.java

```diff
@@ -371,6 +374,13 @@ public RegularFileProperty getWsdl() {
 		return this.allowElementRefs;
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public Property<String> getEncoding() {
```

Comment:
This should be first defined in the `Option` interface with a Javadoc: Encoding to use for generated sources
Then `@Override` here in `WsdlOption`

Reply:
Oh, yes, sorry, I missed that one. I added it now.

BTW: Thanks for creating and sharing this great plugin. I was using nilsmagnus/wsdl2java before, but I've the feeling, the project is abandoned. I like how everything just works with your plugin.